StripePayments/StripePayments/Source/API Bindings/Models/PaymentMethods/STPPaymentMethodEnums.swift

```diff
@@ -281,13 +281,24 @@ extension STPPaymentMethodType {
         switch self {
         // Payment methods such as AmazonPay and other app-to-app redirects that bypass the "redirect trampoline" to give a more seamless user experience for app-to-app.
         // However, when returning to the merchant app in this scenario, the intent often isn't updated instantaneously, requiring polling for intent status updates.
-        case .amazonPay, .cashApp:
+        case .amazonPay, .cashApp, .swish:
             return true
         default:
             return false
         }
     }
 
+    var pollingInterval: TimeInterval {
```

Comment:
Ideally we shouldn't have `pollingInterval` if `requiresPolling` is false. Otherwise it introduces bad states like a PM that requires polling but has a polling interval of 0, etc.

How about 
```
extension STPPaymentMethodType {
  struct PollingRequirement {
    var pollingInterval: TimeInterval
  }
  /// If non-nil, Intents with this PM type do not update immediately after the next action is handled and require us to poll and this property contains the information needed to poll.
  var pollingRequirement: PollingRequirement? {
     switch self { ... }
  }
}
```

Reply:
Ah great suggestion. I've added this. It's a little weird w/ cards since there is an additional condition that determines if polling is required (web-based 3DS2), lmk what you think!